docs/guides/caching.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Caching in Cortex"
+linkTitle: "Caching in Cortex"
+weight: 1
+slug: caching
+---
+
+# Caching in Cortex
+
+Correctly configured caching is important for a production-ready Cortex cluster.
+Cortex has many opportunities for using caching to accelerate queries and reduce cost.  Cortex can use a cache for:
```

Comment:
Please add a new line after this line, otherwise the markdown list is not rendered:

![Screen Shot 2020-03-03 at 10 33 02](https://user-images.githubusercontent.com/1701904/75762064-5ee87380-5d3a-11ea-8f4b-9783188c661c.png)